providers/fcontext.rb

```diff
@@ -0,0 +1,50 @@
+# Support whyrun
+def whyrun_supported?
+  true
+end
+
+use_inline_resources
+
+# Create if doesn't exist, do not touch if fcontext is already registered
+action :add do
+  escaped_file_spec = Regexp.escape(new_resource.file_spec)
+  noregex_file_spec = new_resource.file_spec.split(/(\(\/\.\*\)\?|\.\*)$/)[0]
```

Comment:
I saw I have `find -regex` on my machine.
I think we can use something like `find $(<safe path you extracted>) -regex $(entire expression)`
I'm just afraid of applying `restorecon` on other files and causing side effects.
Your opinion?